scorecard/profile_data/indicators/capital_budget_spending.py

```diff
@@ -48,6 +47,35 @@ class CapitalBudgetSpending(SeriesIndicator):
             "100",
         ],
     }
+    formula_v2 = {
+        "text": "= ((Actual Capital Expenditure - Budgeted Capital Expenditure) / Budgeted Capital Expenditure) * 100",
+        "actual": [
+            "=",
+            "(",
+            "(",
+            {
+                "cube": "capital_v2",
+                "item_description": "capital type code NEW, RENEWAL, UPGRADING",
+                "amount_type": "AUDA",
+            },
+            "-",
+            {
+                "cube": "capital_v2",
+                "item_codes": ["4100"],
```

Comment:
same here https://openup.gitbook.io/municipal-money/scorecard-content/citizen-scorecard#spending-of-capital-budget

Reply:
the logic here is that 4100 was the total row in v1

v2 doesn't have a total row - we're using the total of all rows, for the capital types new, renewal and upgrading (not repairs)